mchf-eclipse/drivers/ui/ui_configuration.c

```diff
@@ -513,6 +528,17 @@ uint16_t UiConfiguration_WriteConfigEntryData(const ConfigEntryDescriptor* ced_p
     case ConfigEntry_Int16:
         retval = UiWriteSettingEEPROM_Int16(ced_ptr->id,*(int16_t*)ced_ptr->val_ptr,ced_ptr->val_default);
         break;
+    case ConfigEntry_Int32:
```

Comment:
Hi, I am not a fan of 32bit config variable approach as implented here.  The reason for not liking it is that we use 2 locations but only reference one in the configuration data description. This calls for trouble. It is now much easier to create "shared use" of configuration variables. This requires careful inspection of every commit in ui_configuration{.c/.h}, i.e. extra work. I couldn't come up with a more clever approach to handle this risk when I refactored the ui_configuration that is why I left this and the reading of arrays out.

Question is: Can we implement a backward compatible approach which is safe to use and evolve and not to complex to use when defining new entries?